tricorders/rust/src/dependencies/mod.rs

```diff
@@ -29,9 +29,6 @@ impl DependencyManager {
     }
 
     pub async fn prepare(&self) -> Result<(), DependencyManagerError> {
-        self.mark_as_loading();
```

Comment:
This sounds like the reason why the tests broke. These function is not doing anything now.